src/ssc_joystick_ros.cpp

```diff
@@ -29,9 +29,11 @@ using namespace AS;  // NOLINT
 
 double joy_fault_timeout = 0.0;
 
+bool engage_speed_module = 0;
+bool engage_steering_module = 0;
+
```

Comment:
Is there a reason we are using numbers for these instead of `bool`s? If there is, we should change the below. See next comment.

Reply:
I was thinking those were defined as some form of int (uint8_t or something), not bool.  I didn't look closely enough.

I don't remember now if/how the readJsonWithError works with bool, that's the only reason I was thinking uint8_t instead of bool.

Reply:
I remember it failing with bool when I testing something else recently. That's why I had used numbers.

Reply:
And I guess I'm wrong about the implicit conversion being problematic. See https://stackoverflow.com/questions/5369770/bool-to-int-conversion